lib/dex-oracle/driver.rb

```diff
@@ -155,6 +135,8 @@ def get_driver_dir
       stdout = adb("shell -x ls #{dir}")
       next if stdout == "ls: #{dir}: Permission denied"
       next if stdout == "ls: #{dir}: No such file or directory"
+      stdout = adb("shell ls #{dir}")
+      next if stdout == "opendir failed, Permission denied"
```

Comment:
I don't remember what shell -x does and I'm hesitant to remove it without understanding.